src/libraries/Microsoft.PowerFx.Interpreter/RecalcEngine.cs

```diff
@@ -167,6 +168,12 @@ public async Task<FormulaValue> EvalAsync(string expressionText, CancellationTok
             // - Some Symbols are metadata-only (like option sets, UDFs, constants, etc)
             // and hence don't require a corresponnding runtime Symbol Value. 
             var symbolsAll = ReadOnlySymbolTable.Compose(runtimeConfig?.GetSymbolTableSnapshot(), symbolTable);
+            var culture = runtimeConfig?.GetService<CultureInfo>();
+
+            if (options == null && culture != null)
+            {
+                options = new ParserOptions() { Culture = culture };
+            }

```

Comment:
I don't think we should do this. 

ParserOptions is for parsing at design-time. 
SymbolValues is for executing the expression at runtime. 

These are different concepts and we should keep them separate.